src/trezor/ui/text.py

```diff
@@ -102,17 +102,30 @@ class Text(ui.LazyWidget):
     def __init__(self,
                  header_text: str,
                  header_icon: bytes,
-                 *content: list,
+                 *content: str,
                  new_lines: bool = True,
                  max_lines: int = TEXT_MAX_LINES,
                  icon_color: int = ui.ORANGE_ICON):
         self.header_text = header_text
         self.header_icon = header_icon
-        self.content = content
+        self.content = list(content)
         self.new_lines = new_lines
         self.max_lines = max_lines
         self.icon_color = icon_color
 
+    def type(self, *content):
```

Comment:
Maybe `normal` would be better choice than `type` considering this is what it does right now?

Reply:
👍